src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -47,12 +47,12 @@ object DatabaseConversions {
         Tables.AccountsRow(
           accountId = id.id,
           blockId = hash.value,
-          manager = manager,
+          manager = manager.value,
```

Comment:
Were the changes in this file due to the schema being wrong/optional fields?

Reply:
I added a wrapping class definition, i.e.
`manager: String` -> `manager: PublicKeyHash`

So now to read the pkh string you need to extract the `.value` 🙂